src/company.cpp

```diff
@@ -0,0 +1,168 @@
+#include "company.h"
+#include <vector>
+#include <algorithm>
+#include <iostream>
+#include <set>
+#include <limits.h>
+
+Worker::Worker() {}
+
+Worker::Worker(long long start, long long finish, long long type, long long salary, long long id) {
+	w_start = start;
+	w_finish = finish;
+	w_type = type;
+	w_salary = salary;
+	w_id = id;
+}
+
+long long Worker::get_type() {
+	return w_type;
+}
+
+long long Worker::get_start() {
+	return w_start;
+}
+
+long long Worker::get_finish() {
+	return w_finish;
+}
+
+long long Worker::get_salary() {
+	return w_salary;
+}
+
+bool operator<(const Worker w_1, const Worker w_2) {
+	return w_1.w_start <= w_2.w_start;
+}
+
+std::istream& operator>> (std::istream &is, Worker &w) {
+	is >> w.w_start;
+	is >> w.w_finish;
+	is >> w.w_type;
+	is >> w.w_salary;
+	is >> w.w_id;
+	return is;
+}
+
+Task::Task() {}
+
+Task::Task(long long type, long long cost) {
+	t_type = type;
+	t_cost = cost;
+}
+
+bool operator<(const Task t_1, const Task t_2) {
+	return t_1.t_type <= t_2.t_type;
+}
+
+bool Task::operator==(const Task& other) const {
+	return t_type == other.t_type && t_cost == other.t_cost;
+}
+
+long long Task::get_cost() {
+	return t_cost;
+}
+
+long long Task::get_type() {
+	return t_type;
+}
+
+std::istream &operator>> (std::istream &is, Task &t) {
+	is >> t.t_type;
+	is >> t.t_cost;
+	return is;
+}
+
+std::ostream &operator<< (std::ostream &os, const Task &t) {
+	os << t.t_type;
```

Comment:
t_cost is forgotten?

Reply:
I actually don't know what output  format is best here. Could you specify it maybe?

Reply:
Then may be just remove 'operator<<' till we need it?